tmt/base.py

```diff
@@ -1682,6 +1687,16 @@ def environment(self) -> EnvironmentType:
             return combined
         return self._environment
 
+    def _source_plan_environment_file(self) -> None:
+        """ Add variables from the plan environment file to the environment """
+        if (self.my_run and self.plan_environment_file.exists()
+                and self.plan_environment_file.stat().st_size > 0):
+            # using __wrapped__ to force the reload of the file
```

Comment:
```suggestion
            # Use __wrapped__ to force the reload of the file
```

Just a consistency nitpick.

Reply:
Fixed in 9ecf336.